src/append/rolling_file/mod.rs

```diff
@@ -194,6 +204,8 @@ impl RollingFileAppender {
         RollingFileAppenderBuilder {
             append: true,
             encoder: None,
+            #[cfg(feature = "compound_policy")]
+            roll_on_startup: false,
```

Comment:
If this is only necessary for a compound policy, shouldn't it then be nested in the compound policy configuration?